client/src/java/com/zimbra/client/ZMailbox.java

```diff
@@ -2610,7 +2610,7 @@ public String uploadContentAsStream(String name, InputStream in, String contentT
         int statusCode;
         try {
             MultipartEntityBuilder builder = MultipartEntityBuilder.create();
-            builder.addBinaryBody("upfile", in, ContentType.create(contentType), name);
+            builder.addBinaryBody("upfile", in, ContentType.DEFAULT_BINARY, name);
```

Comment:
Removing `contentType` here leaves it unused in this method. Is losing that information acceptable?

Reply:
Maybe wit the old library it was necessary. I tested various mimes with content type as binary and did not see any issues. I am going to keep it in method signature as it would have a cascading effect on the calling methods. Trying to keep code change to a minimum